src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can we split this change out into a separate PR? The validity of adding debug logging to test cases is a different concern to a code cleanup